README.md

```diff
@@ -37,7 +37,7 @@ To show the SDK in action run the example:
 ```
 mkdir ~/govcd_example
 go mod init govcd_example
-go get github.com/vmware/go-vcloud-director/v2@main
+go get github.com/vmware/go-vcloud-director/v3@main
```

Comment:
Changelog entry for this important PR would be great!

Reply:
Added, thanks!